runtime/cmake/options.cmake

```diff
@@ -132,9 +132,10 @@ option(J9VM_OPT_FRAGMENT_RAM_CLASSES "Transitional flag for the the GC during th
 option(J9VM_OPT_JVMTI "Support for the JVMTI interface")
 option(J9VM_OPT_JXE_LOAD_SUPPORT "Controls if main will allow -jxe: and relocate the disk image for you.")
 
-option(J9VM_OPT_METHOD_HANDLE "Turns on methodhandle support")
+option(J9VM_OPT_METHOD_HANDLE "Enables support for OpenJ9 MethodHandles. J9VM_OPT_OPENJDK_METHODHANDLE should be disabled.")
 option(J9VM_OPT_MODULE "Turns on module support")
 option(J9VM_OPT_MULTI_VM "Decides if multiple VMs can be created in the same address space")
+option(J9VM_OPT_OPENJDK_METHODHANDLE "Enables support for OpenJDK MethodHandles. J9VM_OPT_METHOD_HANDLE should be disabled.")
```

Comment:
Is there code somewhere that will yield an error if both `J9VM_OPT_METHOD_HANDLE` and `J9VM_OPT_OPENJDK_METHODHANDLE` are enabled? Same question for UMA builds?

Reply:
On second thought, perhaps it's not worth worrying about; the code in `OpenJ9.gmk` in the extension repos won't permit that situation.